pkg/pipelines/init.go

```diff
@@ -109,7 +110,7 @@ const (
 // Init bootstraps a GitOps pipelines and repository structure.
 func Init(o *InitOptions, fs afero.Fs) error {
 	exists, err := ioutils.IsExisting(fs, o.OutputPath)
-	if exists {
+	if exists && !o.Overwrite {
```

Comment:
Hmm.. we check for output directory in "init"command  but pipelines.yaml is checked for bootstrap command.   Do we want to do it more consistently?

Reply:
We should be reusing the same checking function in both.

Reply:
A single function checks for both bootstrap and init now.